niworkflows/anat/mni.py

```diff
@@ -131,14 +137,10 @@ def _config_ants(self, ants_settings):
             num_threads=self.inputs.num_threads,
             from_file=ants_settings,
             terminal_output='file',
-            write_composite_transform=True
+            write_composite_transform=True,
+            initial_moving_transform=self.inputs.initial_moving_transform
         )
 
-        if isdefined(self.inputs.initial_moving_transform):
```

Comment:
why don't something like:

```
if init_transform is None and isdefined(...)
```
where `init_transform` is the result of the call to `AffineInitializer.run()` if it was actually run.